From d716192ea9ba5fc082685b42454978459816939a Mon Sep 17 00:00:00 2001
From: ghost <andreas.monzner@dream-property.net>
Date: Thu, 15 Feb 2018 15:14:18 +0100
Subject: [PATCH 5/6] Converter/ServiceInfo: re-add evVideoSizeChanged to
 "Framerate" interesting events to fix not visible framerate on first service
 start

---
 usr/lib/enigma2/python/Components/Converter/ServiceInfo.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/usr/lib/enigma2/python/Components/Converter/ServiceInfo.py b/usr/lib/enigma2/python/Components/Converter/ServiceInfo.py
index 369da77..a3ab5e0 100644
--- a/usr/lib/enigma2/python/Components/Converter/ServiceInfo.py
+++ b/usr/lib/enigma2/python/Components/Converter/ServiceInfo.py
@@ -44,7 +44,7 @@ class ServiceInfo(Converter, object):
 				"TsId": (self.TSID, (iPlayableService.evUpdatedInfo,)),
 				"OnId": (self.ONID, (iPlayableService.evUpdatedInfo,)),
 				"Sid": (self.SID, (iPlayableService.evUpdatedInfo,)),
-				"Framerate": (self.FRAMERATE, (iPlayableService.evVideoFramerateChanged,)),
+				"Framerate": (self.FRAMERATE, (iPlayableService.evVideoSizeChanged, iPlayableService.evVideoFramerateChanged)),
 				"TransferBPS": (self.TRANSFERBPS, (iPlayableService.evUpdatedInfo,)),
 				"HasSubtitles": (self.HAS_SUBTITLES, (iPlayableService.evUpdatedInfo,)),
 			}[type]
-- 
2.1.4

